File: anagram.py
def anagramlength(string1,string2):
    difference = {}

    for letter in string1:
        if letter not in difference:
            difference[letter] = 0
        difference[letter] += 1
        print difference

    for letter in string2:
        if letter not in difference:
            difference[letter] = 0
        difference[letter] -= 1
        print difference
    print sum(abs(n) for n in difference.values())


if __name__ == "__main__":
    a = raw_input()
    b = raw_input()
    anagramlength(a, b)

File: anagram.js
function findAnagramLength(string1, string2) {
    var difference = {};
    for (var i = 0; i < string1.length; i++) {
        if (difference[string1.charAt(i)] == undefined)
            difference[string1.charAt(i)] = 0
        difference[string1.charAt(i)] += 1
    }
    for (var i = 0; i < string2.length; i++) {
        if (difference[string2.charAt(i)] == undefined)
            difference[string2.charAt(i)] = 0
        difference[string2.charAt(i)] -= 1
    }
    var anagramLength = 0;
    Object.keys(difference).forEach(function (key) {
        anagramLength+=Math.abs(difference[key]);
    });
    return anagramLength;
}

console.log(findAnagramLength("qeltuiosytrabycsnvdourrbvarooaqhgdaaoovmkonlnrdcpirjckkxkbnydffklhmzwhngyctnuzpjjislsfleiyfmgkwrskqqu", ""))